cmd/release-controller/types.go

```diff
@@ -106,6 +106,10 @@ type ReleaseConfig struct {
 	// rejected.
 	Verify map[string]ReleaseVerification `json:"verify"`
 
+	// Periodic is a map of short names to verification steps that run based on a cron
+	// or interval timer.
+	Periodic map[string]ReleasePeriodic `json:"periodic"`
```

Comment:
Can you include an example as a response to this comment or in the pr description that describes how you would be using this?  Would help me answer Steve's question above.

Reply:
Steve misunderstood how this is configured. The configuration is pretty much the same as it is for verify jobs:

```
  "periodic": {
    "upgrade":{
       "upgrade":true,
       "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade-4.6"}
     },
     "upgrade-minor":{
       "upgrade":true,
       "upgradeFrom": "PreviousMinor",
       "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade-4.5-stable-to-4.6-ci"}
     },
  }
```

I am unsure if it should be a map though. This implementation doesn't change the name of the periodic like `verify` does, so it may be better to change it to an array.